docs/tutorials/earthquake_detection.ipynb

```diff
@@ -172,7 +187,7 @@
    "source": [
     "datamodule.setup('fit')\n",
     "datamodule.setup('test')\n",
-    "sample = datamodule.val_dataset[500]\n",
+    "sample = datamodule.val_dataset[min(500, len(datamodule.val_dataset) - 1)]\n",
```

Comment:
Can we instead pick sample 0 or 1 so that the code is simpler?

Reply:
```suggestion
    "sample = datamodule.val_dataset[0]\n",
```